doc/cygnus-ngsi/installation_and_administration_guide/grouping_rules.md

```diff
@@ -0,0 +1,56 @@
+#<a name="top"></a>Grouping Rules
+Grouping rules are an advanced global feature of Cygnus. It is global because it is available for all NGSI sinks.
```

Comment:
Last refactor :) Just reordering the paragraphs:

Grouping rules are an advanced global feature of Cygnus. It is global because it is available for all NGSI sinks. They allow for changing the notified <b>FIWARE service path</b> and the concatenation of <b>entity ID</b> and <b>entity type</b> (this is called the destination).

Such rules are just a Json within a configuration file (that can be instantiated from `grouping_rules.conf.template`) located at `conf/`. he following code is just an example: